diff --git a/setup.py b/setup.py
index 435397d..e27586c 100644
--- a/setup.py
+++ b/setup.py
@@ -48,6 +48,7 @@ setuptools.setup(
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
         'Operating System :: POSIX',
         'Operating System :: MacOS :: MacOS X',
         'Operating System :: Microsoft :: Windows',
diff --git a/tests/test_map.py b/tests/test_map.py
index 8d629f8..0b464cf 100644
--- a/tests/test_map.py
+++ b/tests/test_map.py
@@ -1294,7 +1294,7 @@ class BaseMapTest:
             self.assertTrue(isinstance(uh, self.Map))
             self.assertEqual(h, uh)

-        with self.assertRaisesRegex(TypeError, "can't pickle"):
+        with self.assertRaisesRegex(TypeError, "can('t|not) pickle"):
             pickle.dumps(h.mutate())

     @unittest.skipIf(sys.version_info < (3, 7, 0), "__class_getitem__ is not available")
